codewiki/src/be/utils.py

```
@@ -59,149 +62,155 @@ async def validate_mermaid_diagrams(md_file_path: str, relative_path: str) -> st
-            sys.stderr = open(os.devnull, 'w')
-            
+            sys.stderr = open(os.devnull, "w")
+
             try:
-                json_output = await parse_mermaid_py(diagram_content)
+                await parse_mermaid_py(diagram_content)
             finally:
                 # Restore stderr
                 sys.stderr.close()
                 sys.stderr = old_stderr
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)